### PR TITLE
[FIX] Fix cublas batch matmul

### PR DESCRIPTION
**Pull request by sxjscience**
Thanks for contributing to TVM!   Please refer to guideline https://tvm.apache.org/docs/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/apache/incubator-tvm/blob/master/CONTRIBUTORS.md#reviewers) by @ them in the pull request thread.

@comaniac The issue is the same as the cblas_batch_matmul issue in https://github.com/apache/incubator-tvm/pull/6699.

### CODE REVIEW

File: python/tvm/topi/cuda/batch_matmul.py [hunk 1]
```diff
@@ -149,6 +149,7 @@ def batch_matmul_cublas(x, y):
 
     y : tvm.te.Tensor
         3-D with shape [batch, N, K]
+    _ : None
```
Comment (comaniac) [inline python/tvm/topi/cuda/batch_matmul.py:152]:
```suggestion
    _ : None
        Output shape argument holder to match the function signature
```

**Review by comaniac — CHANGES_REQUESTED** [review]
@merrymercy the CI failed at the Ansor tutorial. Would you mind take a look?

**Review by comaniac — APPROVED** [review]

**Comment by icemelon9** [comment]
Why do we need this attribute in `batch_matmul`?

**Comment by sxjscience** [comment]
The reason is that the code here always assume that there is an `out_shape` argument.  https://github.com/apache/incubator-tvm/blob/461e75bd5ffaf45a0f270998514d444463d11261/python/tvm/relay/op/strategy/generic.py#L685-L686

**Comment by icemelon9** [comment]
I checked the implementation. I think we can remove the oshape arg in the topi implementation and we can enable broadcasting at batch axis by default. cc @jwfromm 
https://github.com/apache/incubator-tvm/blob/main/python/tvm/topi/nn/batch_matmul.py#L23

**Comment by sxjscience** [comment]
Per offline discussion with @icemelon9 , I will have a separate PR to remove the oshape flag directly.

**Review by icemelon9 — APPROVED** [review]
LGTM

**Comment by icemelon9** [comment]
Thanks @sxjscience @comaniac

**Pull request merged**